subt/tools/draw_wifi.py

```diff
@@ -0,0 +1,85 @@
+import sys
+import os
+import numpy as np
+from matplotlib import pyplot as plt
+
+from osgar.logger import LogReader, lookup_stream_id
+from osgar.lib.serialize import deserialize
+
+
+def draw_wifi(logs_dir):
+    path_logs = []
+    if os.path.isdir(logs_dir):
+        sub_dirs = os.listdir(logs_dir)
+        for robot_dir in sub_dirs:
+            robot_dir_path = os.path.join(logs_dir, robot_dir)
+            if os.path.isdir(robot_dir_path):
+                logs = os.listdir(robot_dir_path)
+                for log_name in logs:
+                    path_logs.append(os.path.join(robot_dir_path, log_name))
+            else:
+                path_logs.append(robot_dir_path)  # logs_dir is for one specific robot
+    else:
+        path_logs.append(logs_dir)
+
+    wifi_pose = []
+    co2_pose = []
+    for path_log in path_logs:
+        print(path_log)
+        try:
+            only_stream_wifi = lookup_stream_id(path_log, "wifi.wifiscan")
+            only_stream_co2 = lookup_stream_id(path_log, "gas_detector.co2")
+            only_stream_pose = lookup_stream_id(path_log, "app.pose2d")
```

Comment:
this could be later optional parameter (on SubT we used `rosmsg.pose2d` at the end ... later, not now.